src/Extension/DatagridExtension.php

```diff
@@ -20,7 +20,7 @@
  * @author Sebastiaan Stok <s.stok@rollerscapes.net>
  * @author Norbert Orzechowicz <norbert@fsi.pl>
  */
-class DatagridExtension extends \Twig_Extension
+class DatagridExtension extends \Twig_Extension implements \Twig_Extension_InitRuntimeInterface
```

Comment:
That will pass it to every method call, and would require to do additional checks for every call.
For now this fixes the warning, but in the long run I have some plans to rewrite the internals of this system :+1: